LICENSE-LGPL-3.txt

```diff
@@ -1,7 +1,7 @@
                    GNU LESSER GENERAL PUBLIC LICENSE
                        Version 3, 29 June 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
```

Comment:
I was for a minute sceptical to this change, because of changing in a license file, but https://www.gnu.org/licenses/lgpl-3.0.en.html has the same, so it's fine.